mobile/ios/ios-device.ts

```diff
@@ -322,11 +328,19 @@ export class IOSDevice implements Mobile.IIOSDevice {
 		}).future<void>()();
 	}
 
-    public debug(packageFile: string, packageName: string): IFuture<void> {
-        return (() => {
-            this.$errors.fail({formatStr:"this will come in a future version", suppressCommandHelp: true});
-        }).future<void>()();
-    }
+	public debug(packageFile: string, packageName: string): IFuture<void> {
+		return (() => {
+
+			var debugging: IOSDeviceDebugging = this.$injector.resolve(IOSDeviceDebugging, {
```

Comment:
I think current code is fine as it passes specific values to IOSDeviceDebugging constructor.